test/integration/http_integration.cc

```diff
@@ -279,6 +278,7 @@ void HttpIntegrationTest::cleanupUpstreamAndDownstream() {
   if (codec_client_) {
     codec_client_->close();
   }
+  fake_upstream_connection_.reset();
```

Comment:
Should this be moved under the `if (fake_upstream_connect_) {` scope above? It's correct as is, but more readable there.

Reply:
Oh yeah, definitely, thanks; done.